packages/load-script/test/dom-operations.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { createScriptElement } from '../src/dom-operations';
+
+describe( 'loadScript/dom-operations', () => {
+	describe( 'createScriptElement()', () => {
+		test( 'createScriptElement creates script tag', () => {
+			const url = 'https://example.com/';
+			const script = createScriptElement( url );
+
+			expect( script.src ).toBe( url );
+			expect( script.type ).toBe( 'text/javascript' );
+			expect( script.async ).toBe( true );
+		} );
+
+		test( 'createScriptElement creates script tag with optional arguments', () => {
+			const url = 'https://example.com/';
+			const args = { id: 'scriptId', async: false };
+
+			const script = createScriptElement( url, args );
+
+			expect( script.src ).toBe( url );
+			expect( script.id ).toBe( args.id );
+			expect( script.async ).toBe( args.async );
```

Comment:
This fails, and I don't know why. Any idea @tyxla? It does work in the browser console if I create a script tag, set `async` to true,  then call `removeAttribute( 'async' )`

Reply:
I'm assuming it's due to some weird `jsdom` heuristics around element attribute removal. I've added a suggestion above as to how to address it.